src/filefingerprint.cpp

```diff
@@ -26,6 +26,12 @@
 #include "mega/utils.h"
 
 namespace mega {
+long long abs(long long n)
```

Comment:
Good catch, but this might better go to utils.h & utils.cpp

Reply:
moved, thanks.  I was worried it would eclipse other overloads in the mega namespace, but it turns out this is the only overload of the function that we use.  If we need others later we can bring them into the namespace with 'using'